tests/cypress/tests-e2e/featurePreSelection.cy.js

```diff
@@ -0,0 +1,52 @@
+/// <reference types="cypress" />
+
+describe('Testing the feature selection in the URL', () => {
+    function checkFeatures(featuresIds) {
+        cy.readStoreValue('state.features.selectedFeatures').then((features) => {
+            cy.wrap(features.length).should('be.equal', featuresIds.length)
+            features.forEach((feature) => {
+                cy.wrap(featuresIds.includes(feature._id)).should('be.true')
```

Comment:
Do not use private `_id` but `id` instead, will be more robust the day we get rid of the private prop (which will happen for sure)

Reply:
`.id` returns a mix between the layer id and the feature id (for example : `my.layer.id-8835` )
`._id` returns only the feature id
right now, I'm using the one which returns what I want :)